src/common/constants.ts

```diff
@@ -161,7 +161,9 @@ export class Constants {
     public static updateSimulatorMsg = "Update your 'iotedgehubdev' tool to the latest for the best experience for IoT Edge Simulator.";
     public static failedInstallSimulator = "Failed to install 'iotedgehubdev' tool because of error:";
     public static outputNoSimulatorMsg = "Cannot execute command since 'iotedgehubdev' is not installed. Please install it first for IoT Edge Simulator.";
-    public static installStandaloneFailedMsg = "Failed to install 'iotedgehubdev' tool, please check the output channel for detailed error message.";
+    public static outputSimulatorIsInstallingMsg = "'iotedgehubdev' is installing now, please wait for installing to complete.";

```

Comment:
wait for installation

Reply:
How about this? 
'iotedgehubdev' is being installed now, and please wait for the installation.

Reply:
ok